CHANGELOG.md

```diff
@@ -37,6 +37,9 @@ This command may still be used to show results of a test report. But it is advis
 
 ### Improvements
 
+* Do not print stack trace when a report is in an incompatible version
```

Comment:
Minor nitpick but shouldn't the comments end with a period to maintain consistency? Otherwise looks good.

Reply:
I always forget these these! Thanks :)